modules/snooker/src/main/ts/ephox/snooker/api/Sizes.ts

```diff
@@ -45,7 +43,7 @@ const getUnit = (newSize: string): 'px' | '%' => {
 
 // Procedure to resize table dimensions to optWidth x optHeight and redistribute cell and row dimensions.
 // Updates CSS of the table, rows, and cells.
-const redistribute = (table: SugarElement, optWidth: Optional<string>, optHeight: Optional<string>, tableSize: TableSize): void => {
+const redistribute = (table: SugarElement<HTMLTableElement>, optWidth: Optional<string>, optHeight: Optional<string>, _tableSize: TableSize): void => {
```

Comment:
Why not remove the argument altogether if it's unused?

Reply:
I was trying not to make a breaking change when doing this, but ended up having to do that later anyways. As such I might as well remove this now.

Reply:
This actually required removing in quite a number of places, so the change for it is rather big but it's been done now.